LunarVim/LunarVim
# feat(installer): allow customizing NVIM_APPNAME

# Description

The installer can now make use of `NVIM_APPNAME` if set.

| variable | default value |
|----------|---------|
| `NVIM_APPNAME` | `lvim` |
| `LUNARVIM_CONFIG_DIR` | `XDG_CONFIG_HOME/NVIM_APPNAME` |
| `LUNARVIM_CACHE_DIR` | `XDG_CACHE_HOME/NVIM_APPNAME` |
| `LUNARVIM_RUNTIME_DIR` | `XDG_DATA_HOME/lunarvim`* |


* __this was kept as-is to avoid having to migrate existing installations__

## How Has This Been Tested?




- back up `~/.local/bin/lvim`
- run command `bash utils/installer/install_bin.sh` with the default values
- check that `lvim` still launches correctly
